### ISSUE TITLE
Embedding a message in Messenger.

### ISSUE THREAD

**Issue by Sharpz7** [carpedm20/fbchat#283]
I was wondering of there is a way to make links sent from messenger, embedded, so that it shows a description and thumbnail of a video for example.

Is there a way to do this with client.send() ?

**Comment by madsmtm** [comment]
No, this is not possible with the API, I'll add it to the list ;)

**Comment by stilett0** [comment]
I went to automate sending a URL (in this case, to a Facebook Event) via fbchat with the expectation that it would embed what Facebook thought was a page thumbnail and synopsis of the description for the URL. It did not. Dug around until I discovered it was not yet implemented and this was the ticket for it.
Please consider this a vote for the feature's support, and I would be interested in being notified if/when it is implemented.
